students/SarahR/Session02/series.py

```diff
@@ -0,0 +1,146 @@
+def fibonacci(n):
+    """ (int) -> int
+
+    Parameters:
+    	n: n'th number of the Fibonacci sequence
+
+    Compute the nth Fibonacci number
+
+    """
+    # first two terms
+    n1 = 0
+    n2 = 1
+    nth = int
+    count = 2
+
+    # check validity of n
+    if n < 0:
+        print("Please enter a positive integer")
+	# No calculations for the first 2 elements of the sequence
+    elif n <2:
+        if n == 0:
```

Comment:
Since you return 0 if n == 0 and 1 if n == 1, a quick way to do this is to just return n.